ccmlib/scylla_cluster.py

```diff
@@ -146,7 +146,7 @@ def start_nodes(self, nodes=None, no_wait=False, verbose=False, wait_for_binary_
             for old_node, mark in marks:
```

Comment:
just for readability, to show this isn't used 
```suggestion
            for old_node, _ in marks:
```